recipes/scelestial/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -x -e
+
+mkdir -p ${PREFIX}/bin/ bin/
+rm -f bin/*
+
+if [ "$(uname)" == "Linux" ]; then
```

Comment:
I did not see a different compile method for MacOS and Linux. These if statements for darwin and linux can be removed and compile statements could be written ones.

Reply:
I fixed it. Would you please reconsider it?